.github/workflows/draw-zmk.yml

```diff
@@ -49,11 +49,26 @@ on:
         default: ''
         required: false
         type: string
+      destination:
+        description: 'Add the output files to a commit, as artifacts or both, values: `commit`, `artifact`, `both`'
+        default: 'commit'
+        required: false
+        type: string
+      artifactname:
+        description: 'Define the name the drawings are stored as an artifact'
+        default: 'drawings'
+        required: false
+        type: string
+    outputs:
+      drawings-id:
+        description: 'Archive with keymap in yaml and drawing in svg formats'
+        value: ${{ jobs.draw.outputs.drawings-id }}
 
 jobs:
   draw:
     runs-on: ubuntu-latest
-
+    outputs:
+      drawings-id: ${{ steps.artifact-upload-step.outputs.artifact-id }}
```

Comment:
Is there a need to suffix these with `-id`? They are scoped to `...outputs` IIUC.

Reply:
I think basically this is unnecessary and can be removed if you like. I first thought, I would need the artifact ID for later retrieving it from the second job. I was wrong, the second job can work just fine on the artifact name itself. So it would only be of value for programmers, who are using it in different scripts, where they then construct the REST API URL for the artifacts.

I think we can drop these two lines completely

Reply:
Then let's remove the suffixes, maybe the output will come handy later for someone.